test/extended/apiserver/tls.go

```diff
@@ -62,3 +177,20 @@ var _ = g.Describe("[sig-api-machinery][Feature:APIServer]", func() {
 
 	})
 })
+
+func waitForKASIncrementedRevision(ctx context.Context, operatorClient operatorv1client.Interface, name string, currentRevision int32) (int32, error) {
```

Comment:
I would remove this function and just use `WaitForAPIServerToStabilizeOnTheSameRevision`